src/index.ts

```diff
@@ -1,81 +1,128 @@
+  /**
+   * Returns the current logger level.
+   *
+   */
+  public getLevel () {
+    let level: LogLevel | null = null
+
+    for (const [name, number] of Object.entries(logger.levels)) {
+      if (logger.getLevel() === number) {
+        level = name as LogLevel
+        break
+      }
     }
+    return level
   }
 
-  return JSON.stringify(data)
-}
+  /**
+   * Generates an output log with severity level DEBUG.
+   *
+   * @param {String} message is the name of log event(i.e. request_started, request_finished and etc).
+   * @param {object} payload is a JSON object that represents the content of the event.
+   */
+  public debug (message: string, payload: Object): void {
+    logger.debug(this._buildPayload(LogLevel.DEBUG, message, payload))
+  }
 
-/**
- * Logs an info log.
- *
- * @param message is a JSON object that represents the body of the event.
- * @param event  is the name of request event (request_started, request_finished and etc)
- * @param messageEvent is the name of the system event (BookingFound, BookingNotFound and etc). The default value is 'request'.
- */
-// prettier-ignore
-function info (message: unknown, event: string, messageEvent: string): void {
-  console.info(createMessage(message, event, messageEvent, LogLevel.INFO))
-}
+  /**
+   * Generates an output log with severity level INFO.
+   *
+   * @param {String} message  is the name of log event(i.e. request_started, request_finished and etc).
+   * @param {object} payload is a JSON object that represents the content of the event.
+   */
+  public info (message: string, payload: Object): void {
+    logger.info(this._buildPayload(LogLevel.INFO, message, payload))
+  }
 
-/**
- * Logs a warn log.
- *
- * @param message is a JSON object that represents the body of the event.
- * @param event  is the name of request event (request_started, request_finished and etc)
- * @param messageEvent is the name of the system event (BookingFound, BookingNotFound and etc). The default value is 'request'.
- */
-// prettier-ignore
-function warn (message: unknown, event: string, messageEvent: string): void {
-  console.warn(createMessage(message, event, messageEvent, LogLevel.WARN))
-}
+  /**
+   * Generates an output log with severity level WARN.
+   *
+   * @param {String} message  is the name of log event(i.e. request_started, request_finished and etc).
+   * @param {object} payload is a JSON object that represents the content of the event.
+   */
+  public warn (message: string, payload: Object): void {
+    logger.warn(this._buildPayload(LogLevel.WARN, message, payload))
+  }
+
+  /**
+   * Generates an output log with severity level ERROR.
+   *
+   * @param {String} message  is the name of log event(i.e. request_started, request_finished and etc).
+   * @param {object} payload is a JSON object that represents the content of the event.
+   */
+  public error (message: string, payload: Object): void {
+    logger.error(this._buildPayload(LogLevel.ERROR, message, payload))
```

Comment:
[Suggestion]
We should have a `error` parameter as well.
The message field isn't really the _"log event(i.e. request_started, request_finished and etc)."_. It is the message, and might also be _the log event_.

```suggestion
   * @param {String} message is the core information of your log entry. By reading it anyone should understand what the log entry is about.
   * @param {object} payload is a JSON object with all structured fields to be added to the log.
   * @param {String} error is a description of the error.
   */
  public error (message: string, error: string, payload: Object): void {
    logger.error(this._buildPayload(LogLevel.ERROR, message, { error, ...payload }))
```